config/templates/cluster-deployment-customimageset.yaml

```diff
@@ -106,7 +106,7 @@ objects:
       machineCIDR: "10.0.0.0/16"
       clusterNetworks:
         - cidr: "10.128.0.0/14"
-          hostSubnetLength: 9
+          hostSubnetLength: 23
```

Comment:
cc @jhernand the installer is now more strict about this value, I incremented to match the installer's defaults, if you are passing 9 from UHC install jobs will start erroring against hive-stage once this merges.